### PR TITLE
fixed update qubit frequency after Ramsey

### PR DESCRIPTION
**Pull request by DavidSarlle**
This PR fixes issue #337 returning only delta_phys in qibocal and updating the qubit freq accordingly in qibolab (https://github.com/qiboteam/qibolab/pull/426)

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

**Review by andrea-pasquale — APPROVED** [review]
Thanks @DavidSarlle.
It is working as expected. As soon as qiboteam/qibolab#426 is merged we can also merge this one.

**Comment by DavidSarlle** [comment]
> I think making this many changes on both qibocal and qibolab in order to account for a wrong sign is complete overkill. As this is a change that will be reverted once the discrepancy is fixed. I think acknowledging it in the code, and just locally changing the sign until we reach a conclusion on how to deal with relative phases is better.

The changes are already done. This is a temporary fix, while we understand who is dealing in the wrong way with the relative phase or even if it is possible to modify it.

**Comment by andrea-pasquale** [comment]
Although this PR is quite simple, I think that especially the PR in qibolab will add several things that we will remove soon.
I've modified #339 by leaving some instructions for qblox. It should be as easy as commenting or uncommenting a line for the time being.
I know that I said that this was ok in #337, but as I said, especially the PR in qibolab seems to be too much complicated.
@DavidSarlle when you have time could you please test #337 to see if everything works as expected for qblox?

**Comment by andrea-pasquale** [comment]
Personally I still prefer @igres26 option given that it is just a single line, and at least from the lab side (by this I mean people that will actually run calibration routines) people are already asking me to make the automatic update mechanism optional.  I also think that the fix in the qblox driver should be quite easy to implement.
I would like to hear the opinion of other people before taking a final decision.
@stavros11 @scarrazza @AleCandido @Edoardo-Pedicillo @aorgazf thoughts?

File: src/qibocal/protocols/characterization/ramsey.py [hunk 1]
```diff
@@ -32,12 +32,12 @@ class RamseyParameters(Parameters):
 class RamseyResults(Results):
     """Ramsey outputs."""
 
-    frequency: Dict[List[Tuple], str] = field(metadata=dict(update="drive_frequency"))
-    """Drive frequency for each qubit."""
+    delta_frequency: Dict[List[Tuple], str] = field(
```
Comment (AleCandido) [inline src/qibocal/protocols/characterization/ramsey.py:35]: Completely unrelated to the PR: the type hint is simply wrong. `List` is unhashable.

```py
[ins] In [1]: {[]: 0}
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In [1], line 1
----> 1 {[]: 0}

TypeError: unhashable type: 'list'
```

**Comment by DavidSarlle** [comment]
As we have refactored qibolab, this PR is obsolete. But the bug is still present on main. Befero closing this PR and open a new one, we need to decide what is the approach to fix the delta_phys sign fitted from Ramsey experiment.
I still prefer to calculate the corrected qubit freq in the platform, instead of passing as a parameter in the qibocal action runcard the sign. Please @andrea-pasquale and @igres26 let me know what do you want to do.

After deciding, I will close both PRs (https://github.com/qiboteam/qibocal/pull/341 and https://github.com/qiboteam/qibolab/pull/426) and create new ones with the latest changes in both repos.

**Comment by andrea-pasquale** [comment]
> Please @andrea-pasquale and @igres26 let me know what do you want to do.

I think that it should be fairly easy to check the sign of the relative phase using ro optimization over ro frequency

https://github.com/qiboteam/qibocal/blob/63484ee68322104d1e9bd87fe9b47edb2a5ff00c/src/qibocal/calibrations/characterization/ro_optimization.py#L13

which @Edoardo-Pedicillo is porting to the new layout in #392.
Next week we could test this routine with all instruments to standardize the convention about the relative phase.
Until we do I suggest to keep this PR open just in case things go wrong.

**Comment by DavidSarlle** [comment]
Solution implemented in the new qibocal current layout passing the sign of the detunning (+/-) to the Ramsey routine

**Pull request closed without merging**